apps/betterangels/src/app/(private-screens)/add-note/[noteId].tsx

```diff
@@ -21,7 +21,6 @@ import { useLocalSearchParams, useNavigation, useRouter } from 'expo-router';
 import React, { useLayoutEffect, useMemo, useRef, useState } from 'react';
 import { ScrollView, View } from 'react-native';
 import Location from './Location';
-import Mood from './Mood';
```

Comment:
Hi @Davit-BetterAngels 

I found one thing.

I confirmed that the Mood section is removed in Edit view and View-only sections. However, I found the mood related content in the Public Note. I believe we need to remove that too when we don't have the Mood section overall. 


<img width="360" src="https://github.com/user-attachments/assets/9ffaf3cb-aabe-4672-b1e8-aec458b736a2" />

Reply:
Hey @Davit-BetterAngels 
I still see the mood in the public note... I removed my dev app once and reinstalled it and tested again but still see the mood.